docs/proposals/security-context-constraints.md

```diff
@@ -0,0 +1,303 @@
+## Abstract
+
+Security context constraints relate [security contexts](./security_contexts.md) and
+[service accounts](http://docs.k8s.io/design/service_accounts.md).  They are configured for a
+cluster, namespaces, and users within the cluster to govern how a [security context](./security_contexts.md) is established.
```

Comment:
users within the cluster within a namespace

This can be done with a policy object in a namespace that has a selector or a user list.